README.md

```diff
@@ -55,7 +55,7 @@ Finally, restart OpenResty.
 openresty -s reload
 ```
 
-Here provides a completed [example](https://github.com/apache/incubator-shenyu-nginx/blob/main/example/nginx.conf).
+Here provides a completed [example](https://github.com/apache/incubator-shenyu-nginx/tree/main/example).
```

Comment:
once we add examples of nacos and other registers, the word `example` need to change to `examples`?

Reply:
But I think we still keep directory name in singular form.